hydromt_fiat/api/data_types.py

```diff
@@ -81,25 +80,28 @@ class ExposureBuildingsSettings(BaseModel):
     occupancy_type: str
     max_potential_damage: str
     ground_floor_height: Union[str, float]
-    unit: Units
+    gfh_unit: Union[str, Units] = None
+    unit: Union[str, Units]
```

Comment:
does it make sense to have the Union here? Isn't the main purpose of the Units enumerator to only allow these values?

Reply:
This still uses the Union. Should change that as well, right?